corehq/apps/style/static/style/less/legacy/app_manager.less

```diff
@@ -3,6 +3,13 @@
   top: 1.5em;
 }
 
+.app-comment {
+    font-size: 12pt;
+    font-style: italic;
+    padding-bottom: 1em;
+    color: #666;
```

Comment:
Color hex codes shouldn't be hard-coded, perhaps pull a gray from here https://www.commcarehq.org/styleguide/colors/ Looks like those colors are defined in https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/static/style/less/bootstrap3/includes/variables.less

@biyeun feel free to contradict this advice